apps/payments/next/package.json

```diff
@@ -9,8 +9,5 @@
     "stripe": "11.14.0",
     "tslib": "2.5.0",
     "typescript": "^5.3.3"
-  },
-  "scripts": {
-    "start": "next start"
```

Comment:
We don't need this in both `package.json` and `project.json`.

Reference: [https://nx.dev/reference/project-configuration#project-level-configuration-files](https://nx.dev/reference/project-configuration#project-level-configuration-files)

Reply:
Putting it back!